zoom_monitor.py

```diff
@@ -68,21 +75,26 @@ def _open_participants_list(self):
             for outer in self._driver.find_elements(
                     By.CLASS_NAME, "footer-button-base__img-layer"):
                 try:
+                    self.logger.debug(f"trying to find participants in {outer}")
                     # Check if this footer button contains the participants
                     outer.find_element(By.CLASS_NAME, "SvgParticipantsDefault")
                 except NoSuchElementException:
-                    continue # wrong footer element, try the next one
+                    self.logger.debug("participants not present, next...")
+                    continue  # wrong footer element, try the next one
```

Comment:
:+1: Good catch on this formatting, thanks! There should be 2 spaces before an inline comment delimiter.